content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -291,7 +291,8 @@ Given the ordering and execution for init containers, the following rules
 for resource usage apply:
 
 * The highest of any particular resource request or limit defined on all init
-  containers is the *effective init request/limit*
+  containers is the *effective init request/limit*. If any resource has no
+  resource limit specified this is considered as the highest limit.
```

Comment:
> any resource has no resource

Something about the phrasing trips my brain up. Is there an alternative way to state this?

Reply:
Rereading this now, it is really hard to understand what I mean. Updated the PR

Reply:
@anothertobi are you sure you pushed your changes here?

Reply:
```suggestion
  containers is the *effective init request/limit*. If a resource has no
  resource limit specified this is considered as the highest limit.
```

fixed it for you, @jimangel :)